doc/tutorials/04-lattice_boltzmann/04-lattice_boltzmann_part1.ipynb

```diff
@@ -9,11 +9,12 @@
     "#### Before you start:\n",
```

Comment:
<ul><li>Add links to the documentation here, too.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/3893/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>